src/ServiceBouncer/MainForm.cs

```diff
@@ -28,7 +32,10 @@ public MainForm(string machine)
             machineHostname = machine;
             toolStripConnectToTextBox.Text = machineHostname;
             services = new List<ServiceViewModel>();
-            Microsoft.Win32.SystemEvents.SessionSwitch += SessionSwitch;
+            Microsoft.Win32.SystemEvents.SessionSwitch += SessionSwitch;
+            appTerminationTimer.Tick += new EventHandler(TerminateIfInactive);
+            appTerminationTimer.Interval = 60000; 

```

Comment:
When setting an interval in milliseconds, use Timespan.FromMinutes(1).TotalMilliseconds.
Or FromSeconds etc as this is clearer when reading what the value is without needing to do maths.